CMakeLists.txt

```diff
@@ -284,6 +284,14 @@ elseif(UNIX)
     )
     target_compile_definitions(PortAudio PUBLIC PA_USE_COREAUDIO=1)
     set(PKGCONFIG_CFLAGS "${PKGCONFIG_CFLAGS} -DPA_USE_COREAUDIO=1")
+
+    include(CheckIncludeFiles)
+    check_include_files(stdatomic.h HAVE_STDATOMIC_H)
+    if(NOT HAVE_STDATOMIC_H)
+        target_compile_definitions(PortAudio PRIVATE PA_MAC_USE_OSS_MEMORY_BARRIER=1)
+        set(PKGCONFIG_CFLAGS "${PKGCONFIG_CFLAGS} -DPA_MAC_USE_OSS_MEMORY_BARRIER=1")
```

Comment:
I don't think this is definition is needed in the pkgconfig file because pa_memorybarrier.h is only used internally by PortAudio; applications linking to PortAudio don't `#include` this header.

Reply:
Thanks.